app/views/admin/websites/tasks/_bar.html.erb

```diff
@@ -0,0 +1,97 @@
+  <br>
+
+  <% if organizational_website_manager_permissions?(user: current_user) &&
+      website.newly_requested? %>
+    <%= render 'admin/websites/tasks/approval', website: website %>
+  <% elsif website.newly_requested? %>
+    <div class="usa-alert usa-alert--info">
+      <div class="usa-alert__body">
+        <p class="usa-alert__text">
+          This Website is awaiting Approval.
+        </p>
+      </div>
+    </div>
+  <% elsif website.request_denied? %>
+    <div class="usa-alert usa-alert--warning">
+      <div class="usa-alert__body">
+        <p class="usa-alert__text">
+          This Website request was not approved.
+        </p>
+      </div>
+    </div>
+  <% end %>
+
+  <% if website.request_approved? %>
+    <%= link_to develop_admin_website_path(website), class: "usa-button bg-mint full-width", method: :post do %>
+      <span class="fa fa-paint-brush"></span>
+      &nbsp;
+      Begin development
+    <% end %>
+  <% end %>
+
+  <% if website.in_development? %>
+    <%= link_to stage_admin_website_path(website), class: "usa-button bg-accent-cool-dark full-width", method: :post do %>
+      <span class="fa fa-hammer"></span>
+      &nbsp;
+      Stage the site
+    <% end %>
+  <% end %>
+
+  <% if website.staging? %>
+    <%= link_to launch_admin_website_path(website), class: "usa-button bg-green full-width", method: :post do %>
+      <span class="fa fa-rocket"></span>
+      &nbsp;
+      Launch the site to Production
+    <% end %>
+    <br>
+    <br>
+  <% end %>
+
+  <% if website.staging? || website.production? %>
+    <%= link_to redirect_admin_website_path(website), class: "usa-button bg-base-dark full-width", method: :post do %>
+      <span class="fa fa-forward"></span>
+      &nbsp;
+      Redirect this site
+    <% end %>
+    <br>
+    <br>
+  <% end %>
+
+  <% if website.staging? || website.production? %>
+    <%= link_to archive_admin_website_path(website), class: "usa-button bg-base full-width", method: :post do %>
+      <span class="fa fa-archive"></span>
+      &nbsp;
+      Archive
+    <% end %>
+    <br>
+    <br>
+  <% end %>
+
+  <% if website.archived? %>
+  <div class="usa-alert usa-alert--warning margin-bottom-2">
+    <div class="usa-alert__body">
+      <p class="usa-alert__text">
+        This site has been archived.
+      </p>
+    </div>
+  </div>
+  <% end %>
+
+  <% if website.staging? || website.production? || website.redirect? || website.archived? || website.decommissioned? %>
+    <%= render 'admin/websites/tasks/decommission', website: website %>
+  <% end %>
+
+  <% if organizational_website_manager_permissions?(user: current_user) %>
+    <div class="grid-row margin-top-2">
+      <div class="grid-col-12">
+        <%= link_to reset_admin_website_path(website), class: "usa-button usa-button--secondary full-width", method: :post do %>
+        <span class="fa fa-backward"></span>
+        &nbsp;
+        Reset
+        <% end %>
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
Same thing here...

Can we DRY up the view such that we simply have a single component which takes a block for the state transition condition and returns a links to the next possible states?